platform/linux-generic/pktio/stats/ethtool_stats.c

```diff
@@ -131,9 +131,15 @@ static int ethtool_stats(int fd, struct ifreq *ifr, odp_pktio_stats_t *stats)
 		} else if (!strcmp(cnt, "rx_packets")) {
 			stats->in_packets = val;
 			cnts++;
-		} else if (!strcmp(cnt, "rx_ucast_packets")) {
+		} else if (!strcmp(cnt, "rx_unicast_packets")) {
 			stats->in_ucast_pkts = val;
 			cnts++;
+		} else if (!strcmp(cnt, "rx_broadcast_packets")) {
+			stats->in_bcast_pkts = val;
+			cnts++;
+		} else if (!strcmp(cnt, "rx_multicast_packets")) {
+			stats->in_mcast_pkts = val;
+			cnts++;
```

Comment:
It's likely that broad/multicast have alternative names like other counters.

Reply:
Fixed in V7.